copy_tool.py

```diff
@@ -42,6 +55,7 @@ def download(tempdir):
     kafka_path = os.path.join(tempdir, "kafka")
     os.mkdir(kafka_path)
     extract_kafka_to(kafka_path)
+    print(f"downloaded to {tempdir}")
```

Comment:
Can we also add a similar comment after get_connector?

Reply:
I removed all mentions of this file from readme. See pull request description. We are not testing this script, and I want to refrain from putting it in the quickstart because it may introduce a broken devEx.

Reply:
Sure, but the changes I'm asking for are in the file itself.